src/lib/components/TimeSeriesSlider/TimeSeriesSlider.js

```diff
@@ -83,24 +86,24 @@ class TimeSeriesSlider extends React.Component {
       field
     } = sliderLayerObj.layerObj.aggregate.timeseries;
     sliderLayerObj.data = sliderLayerObj.periodData[sliderLayerObj.period[nextIndex]].data;
-    const activeStops = generateStops(sliderLayerObj, field, this.props.dispatch, nextIndex);
-
-    const {
-      primaryLayer
-    } = this.props;
-    if (this.props.layers[primaryLayer].layers) {
-      this.props.layers[primaryLayer].layers.map(i =>
-        nextTimeseries[i].newBreaks = activeStops[3]);
-      this.props.layers[primaryLayer].layers.map(i =>
-        nextTimeseries[i].newColors = [...new Set(sliderLayerObj.colorStops[nextIndex].map(d =>
-          d[1]))]);
-
-      this.props.dispatch(Actions.updateTimeseries(this.props.mapId, nextTimeseries, this.props.timeLayer));
-    } else {
-      nextTimeseries[sliderLayerObj.layerId].newBreaks = activeStops[3];
-      nextTimeseries[sliderLayerObj.layerId].newColors = [...new Set(sliderLayerObj.colorStops[nextIndex].map(d => d[1]))];
-      this.props.dispatch(Actions.updateTimeseries(this.props.mapId, nextTimeseries, this.props.timeLayer));
+    const { layerObj } = sliderLayerObj;
+    if (layerObj.type !== 'chart' && layerObj.property) {
+      const activeStops = generateStops(sliderLayerObj, field, this.props.dispatch, nextIndex);
```

Comment:
Nice, good catch, we definitely don't want to `generateStops` unless we can / have to!